PreferencesWindowController.m

```diff
@@ -42,6 +43,7 @@ - (void)windowDidLoad
     
     [self.pauseOnSleepButton bind:NSValueBinding toObject:defaults withKeyPath:@"values.pauseOnSleep" options:nil];
     [self.pauseOnScreensaverButton bind:NSValueBinding toObject:defaults withKeyPath:@"values.pauseOnScreensaver" options:nil];
+    [self.appearDisabledWhilePaused bind:NSValueBinding toObject:defaults withKeyPath:@"values.appearDisabledWhilePaused" options:nil];
```

Comment:
```suggestion
    [self.appearDisabledWhilePausedButton bind:NSValueBinding toObject:defaults withKeyPath:@"values.appearDisabledWhilePaused" options:nil];
```

Reply:
Oh, I see now! That's really weird, I can't see how I missed it. Thank you!